kde/CMakeLists.txt

```diff
@@ -1,8 +1,3 @@
-if(NOT DEFINED DIGIDOC_EXECUTABLE)
-    set(DIGIDOC_EXECUTABLE qdigidocclient)
-    set(DIGIDOC_ICON qdigidoc-client)
-endif()
-
 configure_file(qdigidoc-signer.desktop.cmake qdigidoc-signer.desktop)
```

Comment:
```suggestion
```
Since there is no reason to configure desktop file can you remove this and rename qdigidoc-signer.desktop.cmake ->  qdigidoc-signer.desktop

Reply:
And later use
install(FILES qdigidoc-signer.desktop ...